source/agora/consensus/Fee.d

```diff
@@ -401,6 +401,33 @@ public class FeeManager
         } ();
     }
 
+    /***************************************************************************
+
+        Calculate the transaction fee and adjust the fee based on the
+        transaction's size measured in bytes.
+
+        The bigger the transaction size is, the smaller the the fee becomes.
+
+        Params:
+            tx = transaction for which we want to calculate the adjusted fee
+            peekUTXO = UTXO finder
+            tot_fee = adjusted fee
+
+        Returns: string describing the error, if an error happened, null otherwise
+
+    ***************************************************************************/
+
+    public string getAdjustedTXFee (in Transaction tx, scope UTXOFinder peekUTXO,
+        out Amount tot_fee) nothrow @safe
```

Comment:
Out of curiosity, what does the `tot` stand for?

Reply:
total -> same name used in getTXFee